students/TacSPx/lesson03/mailroom.py

```diff
@@ -0,0 +1,156 @@
+# ---------------------------------------------------------------------------- #
+# Title: mailroom part 1
+# Description: A program that holds a list of donors and amounts they donated.
+#              Prompt the user to choose 3 menu actions; thank you, report, quit.
+#
+# <05/30/2020>, Created Script
+# ---------------------------------------------------------------------------- #
+
+import sys  # imports go at the top of the file
+
+# Data ---------------------------------------------------------------------- #
+# Task 1: Create data
+# Create a data structure that holds a list of your donors and a history of the amounts they have donated.
+# Have least five donors, with between 1 and 3 donations each, store in the global namespace.
+
+donor_info = [("Jim Zorn", [3772.32, 1201.17]),
+            ("Jermaine Kearse", [877.33]),
+            ("Marcus Trufant", [1563.23, 1043.87, 1.32]),
+            ("K.J. Wright", [21663.23, 300.87, 100432.0]),
+            ("Curt Warner", [663.23, 300.87, 10432.0]),
+            ]
+
+# Processing  --------------------------------------------------------------- #
+# Task 2:  Menu choices
+# Prompt the user to choose from a menu of 3 actions: “Send a Thank You”, “Create a Report” or “quit”.
+###############################################
+#   "Menu of Choices"
+###############################################
+
+def main():
+    '''
+    Menu of Choices:  Thank You, Report, Create a Report
+    :return:
+    '''
+
+    while (True):
+        print("""
+        \n**** XYZ Nonprofit Charity ****\n
+        Menu of Options
+        1) Send a "Thank You"
+        2) Create a Report
+        3) Exit Program
+        """)
+        strChoice = str(input("Which option would you like to perform? [1 to 3] - "))
```

Comment:
You'll want variables (and almost everything else except classes) to be snake_case